### ISSUE TITLE
max() not returning the max integer value in a list

### ISSUE THREAD

**Issue by plajjan** [actonlang/acton#1130]
### Acton Version

0.14.2.20221226.12.1.16

### Steps to Reproduce

```
actor main(env):
    l = [4, 9, 7, 1]
    if max(l, 0) != 9:
        print("max() incorrectly returned:", max(l, 0))
        await async env.exit(0)
```

### Expected Behavior

Get 9 back!

### Actual Behavior

It seems I'm just getting the last value in the list

**Comment by sydow** [comment]
Fixed by #1150

**Comment by plajjan** [comment]
I'm reopening and self assigning to add a unit test to verify correct min and max functionality.
